src/Enhavo/Bundle/NewsletterBundle/Newsletter/NewsletterManager.php

```diff
@@ -176,9 +185,25 @@ public function sendTest(NewsletterInterface $newsletter, string $email)
             ->setFrom($this->from)
             ->setTo($email)
             ->setBody($this->render($newsletter, $this->provider->getTestParameters()));
+
+        if (!empty($newsletter->getAttachments())) {
+            $this->addAttachmentsToNewsletter($newsletter->getAttachments(), $message);
+        }
+
         return $this->mailer->send($message);
     }
 
+    private function addAttachmentsToNewsletter($attachments, \Swift_Message $message) {
```

Comment:
Obviously you adding attachments to message and not to the newsletter itself, so i prefer to name it ``addAttachmentsToNewsletter``